scripts/kfctl.sh

```diff
@@ -112,6 +138,16 @@ if [ "${COMMAND}" == "init" ]; then
       esac
       shift
 	done
+
+  # Download minikube dependencies
+  if [ "${PLATFORM}" == "minikube" ]; then
+    download_kfctl_scripts
+  fi
+
+  source "${DIR}/util.sh"
```

Comment:
Move this back to the top; they will already exist because kfctl.sh doesn't download the scripts.